docs/install.md

```diff
@@ -83,6 +83,20 @@ within a temporary runtime directory.
 ### Ubuntu 14.04:
 You can try to carry on the work done here: <https://gist.github.com/dagelf/99ee07f5638b346adb8c058ab3d57492>
 
+### openSUSE Tumbleweed:
+
+> If you're running transactional server, first chroot into snapshot by
+> issuing `$ transactional-update shell`. It's also recommended that you don't put bees executables into read-write area if you did. 
+
+    $ zypper install libbtrfs-devel uuid-devel
+    $ zypper install -t pattern devel_C_C++
+    $ make
+
+By default `./scripts/beesd.in` isn't compatible with openSUSE. Fix it by editing:
+
+    readonly CONFIG_DIR="# Path to configurations directory; Use absolue path #"
+    readonly bees_bin="# Path to bees executable; Use absolue path #"
```

Comment:
Any reason why we can't fix that in beesd.in or from the top level make?  Something like `make ETC_PREFIX=/path/to/config/dir LIBEXEC_PREFIX=/path/to/libexec`?  Does it need a special `PREFIX` because of the transactional thing?

If not...it should be "absolute".

Reply:
This is pretty much a workaround rather than solution, for those who just want it to work.
And yes this is related to transactional server.
If you want it to have correct prefix then you have to compile it outside of chroot snapshot, introduces more steps to install. (install dev tools, then restart, then compile, then chroot and put executables then restart vs install tools and compile then put executables, then restart)
Compiling with var options will work, but then it won't follow system configurations. Even though it's pretty rare, users may define a different location for /etc. I feel like it's better to let users define the location manually, it's not complicated.

Reply:
I'm not sure if I understand the problem correctly but you can pretty much just use both variables there.

You just need to install with `make DESTDIR=/path/to/package-manager/install-image install`, and then just use the default settings for ETC_PREFIX / PREFIX / LIBEXEC_PREFIX. This is how Gentoo does it: Gentoo internally install to `/var/tmp/portage/bees-VERSIOB/image` but by using DESTDIR:

> https://github.com/gentoo/gentoo/blob/1ae3e0e2c187a73cf53eb7e1d329ab8322abca00/sys-fs/bees/bees-9999.ebuild#L76

This configures the source code, later, the package manager will install with DESTDIR properly set and create a tar from the generated directory structure which includes proper absolute paths for the final installation directory `/`.

I think there's a common misconception to set PREFIX to the install image location if you'd want to create a clean image from it for tar.gz or a package manager - but that is wrong. That is what DESTDIR is for.